### ISSUE TITLE
Document GitHub authentication details

### ISSUE THREAD

**Issue by redcatbear** [exasol/release-robot#17]
## Acceptance Criteria
1. Documentation mentions the credentials file should be set to: `r-- --- ---`
   ```bash
   chmod u-wx,g-rwx,o-rwx "$HOME"/.release-robot/credentials
   ```
2. Documentation lists minimum permissions needed in the OAuth token.

**Comment by redcatbear** [comment]
I'll start my experiments with
* repo_status
* public_repo

**Comment by redcatbear** [comment]
Worked.

**Comment by AnastasiiaSergienko** [comment]
Closed with #23
